src/main/webapp/app/shared/layouts/main/main.component.scss

```
@@ -0,0 +1,4 @@
+.course-overview {
+    background-color: var(--bs-body-bg);
+    padding-top: 0px;
```

```suggestion
    padding-top: 0;
```

or add the bootstrap class `pt-0` to the affected element